### DISCUSSION TITLE
网页端报错：failed get link: ParamFlowException: xxxxxx

### DISCUSSION THREAD

**Discussion by Bssn520** [AlistGo/alist#3599] [Q&A]
网页端播放时报错：  
<img width="878" alt="image" src="https://user-images.githubusercontent.com/96654048/221209777-65cbac16-2fd6-40e1-8e48-2e9579f62a87.png">
但是刷新几次之后又能正常播放了。。。非常疑惑🤔
alist是v3.11.0版本  
复现链接：http://81.70.1.225:5244/%E5%91%A8%E4%B8%80-%E6%98%9F%E8%BE%B0%E5%8F%98

**Comment by balabalazhoucj** [comment]
+1

**Reply by Clydascope** [reply]
同样的问题

**Reply by Bssn520** [reply]
> +1



> 同样的问题

Token被限制了.....换个号吧

**Reply by balabalazhoucj** [reply]
> > +1
> 
> > 同样的问题
> 
> Token被限制了.....换个号吧

换了新的也一样的

**Reply by Bssn520** [reply]
> > > +1
> > 
> > 
> > > 同样的问题
> > 
> > 
> > Token被限制了.....换个号吧
> 
> 换了新的也一样的

我用朋友的Token之后就正常了
